docs/installation.rst

```diff
@@ -113,7 +113,7 @@ are available on the releases_ page for the following platforms:
 * **MacOS**: 10.10 or newer (To avoid signing issues download the file via
   command line **or** remove the ``quarantine`` attribute after downloading:
   ``$ xattr -dr com.apple.quarantine borg-macosx64.tgz``)
-* **FreeBSD**: 10.2 (unknown whether it works for older releases)
+* **FreeBSD**: |freebsd_version| (unknown whether it works for older releases)
```

Comment:
if this is only used at a single place, guess it is easier to just have it literally there, not using a variable.